Update kage_003.txt (#180)
07th-mod/higurashi-console-arcs: Update/kage_003.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500229", 256, TRUE);
	OutputLine(NULL, "「しかし、どうして県境を越えて、この垣内市を？",
			NULL, "\"But, why cross over to another prefecture entirely? Why Kakiuchi City?", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500230", 256, TRUE);
	OutputLine(NULL, "　別に難癖を申し上げるつもりはありませんが、鹿骨市近隣の町村は他にもあったはずです」",

> 			NULL, "Not to saying it was wrong of you to do so, but there must be a couple of places that are closer to Shishibone City.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "Not to say it was wrong of you to do so, but there must be a couple of places that are closer to Shishibone City.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500232", 256, TRUE);
	OutputLine(NULL, "「…はぁ、そういうものなんですか」",

> 			NULL, "\"Huh, so that's how it is...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Huh, so that's how it is...?\"", Line_Normal);
```

	ClearMessage();

//赤坂rvS08/13/321000304.「それにもしも、この災害の背後で組織だった政治的陰謀が本当に動いているとすれば、厳重な情報封鎖とかん口令が敷かれている可能性があります」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000304", 256, TRUE);
	OutputLine(NULL, "「それにもしも、この災害の背後で組織だった政治的陰謀が本当に動いているとすれば、厳重な情報封鎖とかん口令が敷かれている可能性があります」",

> 			NULL, "\"Besides, if this disaster was actually part of a greater political conspiracy, there's always the chance that they've already taken control of the narrative and placed a gag rule on anyone who may know valuable information.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Besides, if this disaster was actually part of a greater political conspiracy, there's always the chance that they've already taken control of the narrative and placed a gag order on anyone who may know valuable information.\"", Line_Normal);
```

	ClearMessage();

//赤坂rvS08/13/321000305.「だとすれば、いずれこの町にもその影響が及んで、捜査がやりにくくなるかもしれない。kvS08/13/321000306.ですから、一刻も早く…」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000305", 256, TRUE);
	OutputLine(NULL, "「だとすれば、いずれこの町にもその影響が及んで、捜査がやりにくくなるかもしれない。",

> 			NULL, "\"If that happens to be true, it's only a matter of time before misinformation spreads to this city, and complicates our investigation even further.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"If that happens to be true, it's only a matter of time before misinformation spreads to this city and complicates our investigation even further.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500233", 256, TRUE);
	OutputLine(NULL, "「えっと、…あの、すみません。",
			NULL, "\"Umm... I'm sorry.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500234", 256, TRUE);
	OutputLine(NULL, "お気を悪くされるかもしれませんが…」",
			NULL, "Please don't take this the wrong way...\"", Line_Normal);
	ClearMessage();

//r南井警部は戸惑ったように小首を傾げながら、慎重に前置きして言った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "南井警部は戸惑ったように小首を傾げながら、慎重に前置きして言った。",

> 			NULL, "With a tilt of her head, Inspector Minai carefully ponders her words before speaking again.", Line_Normal);

Reviewer: ```suggestion
			NULL, "With a tilt of her head, Chief Inspector Minai carefully pondered her words before speaking again.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500237", 256, TRUE);
	OutputLine(NULL, "「それに、自分の職務は犯罪事件を捜査することであって、犯罪自体の存在の有無を確かめることではありません。",

> 			NULL, "\"Besides, as part of the criminal investigations unit, my job's to investigate criminal offenses, not figure out whether a crime happened in the first place.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Besides, as part of the criminal investigations unit, my job is to investigate criminal offenses, not figure out whether a crime happened in the first place.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000308", 256, TRUE);
	OutputLine(NULL, "「…失礼しました。",
			NULL, "\"...I'm sorry for bringing this up then.", Line_WaitForInput);
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000309", 256, TRUE);
	OutputLine(NULL, "つい、気持ちが急いてしまって」",
			NULL, " Seems like I let my feelings get the best of me here.\"", Line_Normal);
	ClearMessage();

//r少々、熱くなりすぎて話が脱線したと、私は咳払いして気を落ち着ける。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "少々、熱くなりすぎて話が脱線したと、私は咳払いして気を落ち着ける。",
			NULL, "Before I knew it, the conversation trailed off and the temperature started getting unreasonably warm, so I cleared my throat while trying to pull myself together.", Line_Normal);
	ClearMessage();

//r南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。
	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。",

> 				NULL, "<size=-2>Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possiblity of a conspiracy surrounding another town's gas disaster, doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);

Reviewer: ```suggestion
				NULL, "<size=-2>Chief Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possibility of a conspiracy surrounding another town's gas disaster, doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```

	} else {
		OutputLine(NULL, "南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。",

> 				NULL, "Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possiblity of a conspiracy surrounding another town's gas disaster, doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);

Reviewer: ```suggestion
				NULL, "Chief Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possibility of a conspiracy surrounding another town's gas disaster, doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。",

> 				NULL, "<size=-2>Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possiblity of a conspiracy surrounding the gas disaster in another town doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);

Reviewer: ```suggestion
				NULL, "<size=-2>Chie Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possibility of a conspiracy surrounding the gas disaster in another town doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```

	} else {
		OutputLine(NULL, "南井警部の見解は保守的ではなく、もっともなことだ。この地域の刑事課に他県のガス災害、そして陰謀云々を語ったところで、職務としてできることは何もない。",

> 				NULL, "Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possiblity of a conspiracy surrounding the gas disaster in another town doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);

Reviewer: ```suggestion
				NULL, "Chief Inspector Minai wasn't purposefully being dismissive, her opinions are completely valid. Just because I discuss the possibility of a conspiracy surrounding the gas disaster in another town doesn't mean that they have to follow up on it, especially when they don't even work in that prefecture.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s08/11/321100229", 256, TRUE);
	OutputLine(NULL, "「…そして、あの祟りの数日後に大災害が起こって、村人全員が亡くなった」",

> 			NULL, "\"...And if so, that means that a few days after the "curse", a massive disaster took place that killed everyone in that village.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...And if so, that means that a few days after the \"curse\", a massive disaster took place that killed everyone in that village.\"", Line_Normal);
```

> 			NULL, "\"...And if so, that means that a few days after the "curse" took place, a massive disaster took place that killed everyone in that village.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...And if so, that means that a few days after the \"curse\" took place, a massive disaster occurred that killed everyone in that village.\"", Line_Normal);
```

//r南井警部は両手を組み、あごを埋めて考え込む。そして空になったカップを見つめながら、静かに切り出した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "南井警部は両手を組み、あごを埋めて考え込む。そして空になったカップを見つめながら、静かに切り出した。",

> 			NULL, "Inspector Minai laced her fingers and pressed them against her chin in contemplation. Then, while gazing at her now empty cup of coffee, she breaks her silence.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Chief Inspector Minai laced her fingers and pressed them against her chin in contemplation. Then, while gazing at her now empty cup of coffee, she broke her silence.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500243", 256, TRUE);
	OutputLine(NULL, "「…自分もあの事件には、どうにも引っかかるものを感じておりました」",

> 			NULL, "\"...You know, that incident has been weighing on mind for some time now.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...You know, that incident has been weighing on my mind for some time now.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500244", 256, TRUE);
	OutputLine(NULL, "「その背後にあるものが雛見沢村の祟りにまつわる可能性があるというのならば、その正体を暴くことは例の焼殺事件に関する手がかりになるかもしれない」",

> 			NULL, "\"If the true culprit behind the curse in Hinamizawa is also involved the great disaster, then acertaining the truth behind everything you've told me should also provide valuble clues to solving the burned corpse incident.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"If the true culprit behind the curse in Hinamizawa is also involved the great disaster, then ascertaining the truth behind everything you've told me should also provide valuable clues to solving the burned corpse incident.\"", Line_Normal);
```

> 			NULL, "\"If the true culprit behind the curse in Hinamizawa is also involved in that great disaster, then acertaining the truth behind everything you've told me should also provide valuble clues to solving the burned corpse incident.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"If the true culprit behind the curse in Hinamizawa is also involved in that great disaster, then ascertaining the truth behind everything you've told me should also provide valuable clues to solving the burned corpse incident.\"", Line_Normal);
```

//r私と大石さんはこの町ではいわば『異邦人』だ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "私と大石さんはこの町ではいわば『異邦人』だ。",
			NULL, "Ooishi-san and I are essentially <i>foreigners</i> here.", Line_Normal);
	ClearMessage();

//r実際に誰がどこにいて、何があるのかといったことは書類上のデータでしか把握していない弱みがあった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "実際に誰がどこにいて、何があるのかといったことは書類上のデータでしか把握していない弱みがあった。",
			NULL, "In all actuality, we fell completely short on any of the particulars regarding this town. Our knowledge only extended as far as the data compiled in official documents, which didn't go beyond information such as where anyone lives or what was around the area.", Line_Normal);
	ClearMessage();

//赤坂rvS08/13/321000315.「私たちには残念ながら、土地勘がありません。kvS08/13/321000316.郷に入っては、のたとえではありませんが、その状態で見当もなく無闇に捜査を行っても、本来求めるべき正確な情報を掴むことは難しいでしょう」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000315", 256, TRUE);
	OutputLine(NULL, "「私たちには残念ながら、土地勘がありません。",

> 			NULL, "\"... we aren't too familiar with this area, unfortunately.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...We aren't too familiar with this area, unfortunately.", Line_WaitForInput);
```

//r実際に誰がどこにいて、何があるのかといったことは書類上のデータでしか把握していない弱みがあった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "実際に誰がどこにいて、何があるのかといったことは書類上のデータでしか把握していない弱みがあった。",

> 			NULL, "Gathering information on whoever lives in this town and what's around the area only through data compiled in official documents on its own has its setbacks.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Gathering information on whoever lives in this town and what's around the area just based on data compiled in official documents has its setbacks.", Line_Normal);
```

			NULL, "\"That's true.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500247", 256, TRUE);
	OutputLine(NULL, "それにこの街の人はわりと淡白な傾向ですから、普通に聞き込みをしても有効な回答を得るのは結構、骨が折れると思います」",
			NULL, "Trust me, I know the people in this town tend to be indifferent about what happens around here, so if you don't ask the right questions, getting the information you want will be pretty much like pulling teeth.\"", Line_Normal);
	ClearMessage();

//r…かつての雛見沢のように排他的な結束がある場所の捜査は、聞き込みなどの情報収集において大きな障害がある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…かつての雛見沢のように排他的な結束がある場所の捜査は、聞き込みなどの情報収集において大きな障害がある。",

> 			NULL, "Investigating such a tightknit community like Hinamizawa is huge a problem for law enforcement. People are fiercely loyal to each other above all, which serverely limits the amount of information we're able to gather there.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Investigating such a tightknit community like Hinamizawa is huge a problem for law enforcement. People are fiercely loyal to each other above all, which severely limits the amount of information we're able to gather there.", Line_Normal);
```

			NULL, "\"That's true.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500247", 256, TRUE);
	OutputLine(NULL, "それにこの街の人はわりと淡白な傾向ですから、普通に聞き込みをしても有効な回答を得るのは結構、骨が折れると思います」",
			NULL, "Trust me, I know the people in this town can be pretty frank about what they think, but if you don't know the right questions to ask they can be difficult enough to make you want to pull your hair out.\"", Line_Normal);
	ClearMessage();

//r…かつての雛見沢のように排他的な結束がある場所の捜査は、聞き込みなどの情報収集において大きな障害がある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…かつての雛見沢のように排他的な結束がある場所の捜査は、聞き込みなどの情報収集において大きな障害がある。",

> 			NULL, "Investigating such a tightknit community as Hinamizawa is huge a problem for law enforcement. People are fiercely loyal to eachother above all, which serverely limits the amount of information we're able to gather there.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Investigating such a tightknit community as Hinamizawa is huge a problem for law enforcement. People are fiercely loyal to eachother above all, which severely limits the amount of information we're able to gather there.", Line_Normal);
```

	ClearMessage();

//r逆に新興の市街によくある、隣人に無関心な雰囲気も警察にとっては厄介な場合が多い。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "逆に新興の市街によくある、隣人に無関心な雰囲気も警察にとっては厄介な場合が多い。",
			NULL, "But on the other hand, the atmosphere in newly founded communities also has its own issues. When people feel completely indifferent toward their neighbors, they might not pay enough attention to even know any meaningful information in the first place.", Line_Normal);
	ClearMessage();

//r各人から集めた情報が非常に曖昧で、正確性に欠けるものとなる場合が多いからだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "各人から集めた情報が非常に曖昧で、正確性に欠けるものとなる場合が多いからだ。",

> 			NULL, "There's been plenty of times where information gained from each witness can be very ambiguous, and in some cases even inaccurate.", Line_Normal);

Reviewer: ```suggestion
			NULL, "There are been plenty of times where information gained from each witness is very ambiguous, and in some cases even inaccurate.", Line_Normal);
```

	ClearMessage();

//赤坂rvS08/13/321000317.「…あと、よそ者の刑事が好き勝手に捜査を行えば、そちらの刑事課さんにとっても色々と問題があるかと思います」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000317", 256, TRUE);
	OutputLine(NULL, "「…あと、よそ者の刑事が好き勝手に捜査を行えば、そちらの刑事課さんにとっても色々と問題があるかと思います」",

> 			NULL, "\"And if people think that we're just a couple of outside police officers sticking our nose into their business, it could pose problems for the local detectives as well.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"And if people think that we're just a couple of outside police officers sticking our noses into their business, it could pose problems for the local detectives as well.\"", Line_Normal);
```
